smileidentity/android
# feat: strict mode

Fix strictmode

## Summary

After in sdk nav, Strict mode wasn't working correctly, this PR fixes that

## Known Issues

N/A

## Test Instructions

- Turn on strict mode you should see the strict mode flow
- Turn off strict mode you should see the older flow

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.
